app/mailers/task_mailer.rb

```diff
@@ -0,0 +1,34 @@
+class TaskMailer < ApplicationMailer
+  def owner_task_created(project, task)
+    @project = project
+    @task = task
+    @owner = ProjectMembership.find_by(project: @project, role: :owner).user
+
+    mail(to: owner.email.pluck(:email))
+  end
+
+  def member_task_created(project, task)
+    @project = project
+    @task = task
+
+    for user in project.users do
+      if user.role == 'member'
+        mail(to: member.email.pluck(:email))
```

Comment:
Maybe it is better to code it smth like:
```suggestion
      mail(to: project.users.where(project_memberships: { role: :member }).pluck(:email))
```
?